play/src/pusher/controllers/IoSocketController.ts

```
@@ -870,24 +869,41 @@ export class IoSocketController {
                                         case "enterChatRoomAreaQuery": {
-                                            await socketManager.handleEnterChatRoomAreaQuery(
-                                                socket,
-                                                message.message.queryMessage.query.enterChatRoomAreaQuery.roomID
-                                            );
-
-                                            answerMessage.answer = {
-                                                $case: "enterChatRoomAreaAnswer",
-                                                enterChatRoomAreaAnswer: {},
-                                            };
-
+                                            try {
+                                                await socketManager.handleEnterChatRoomAreaQuery(
+                                                    socket,
+                                                    message.message.queryMessage.query.enterChatRoomAreaQuery.roomID
+                                                );
+                                                answerMessage.answer = {
+                                                    $case: "enterChatRoomAreaAnswer",
+                                                    enterChatRoomAreaAnswer: {},
+                                                };
+                                            } catch (e) {
+                                                console.warn("Error entering chat room area", e);
+                                                answerMessage.answer = {
+                                                    $case: "error",
+                                                    error: {
+                                                        message: "Error entering chat room area, try again later 🙏",
+                                                    },
+                                                };
+                                            }
```

Why not let this propagate to the catch block above? Is this because you want to reduce the number of Sentry calls? If yes, shouldn't we instead treat the cause?